auslib/db.py

```diff
@@ -1756,20 +1759,39 @@ def __init__(self, db, metadata, dialect):
         AUSTable.__init__(self, db, dialect, scheduled_changes=True, scheduled_changes_kwargs={"conditions": ["time"]})
 
     def getPotentialRequiredSignoffs(self, affected_rows, transaction=None):
-        potential_required_signoffs = []
+        potential_required_signoffs = {}
```

Comment:
When we talked earlier about changing the return type I was under the impression that you'd be changing it for all instances of `getPotentialRequiredSignoffs`, not just this one. Keeping that return type consistent will avoid the need for ugly hacks like two hunks above this (they can be updated to assume dictionaries instead).

Reply:
Oh, now I get it. Sorry about that, I had thought that all we had to do was change `releases.getPotentialRequiredSignoffs` to return dict. Going to work on this ASAP. Thanks.